### ISSUE TITLE
abrt sabotage scylla startup on AMI ami-ac4f69c6

### ISSUE THREAD

**Issue by benoit-canet** [scylladb/scylla#886]
On current AMIs:

        [root@ip-172-30-0-197 ~]# systemctl status scylla-server
        ● scylla-server.service - Scylla Server
           Loaded: loaded (/usr/lib/systemd/system/scylla-server.service; enabled; vendor preset: disabled)
           Active: failed (Result: exit-code) since Mon 2016-02-08 18:57:53 UTC; 7s ago
          Process: 722 ExecStartPre=/usr/bin/sudo -E /usr/lib/scylla/scylla_prepare (code=exited, status=1/FAILURE)

        Feb 08 18:57:51 ip-172-30-0-197 sudo[722]: ---> Package python-pyasn1.noarch 0:0.1.6-2.el7 will be installed
        Feb 08 18:57:52 ip-172-30-0-197 sudo[722]: --> Processing Conflict: scylla-server-0.17-20160208.b2eb081.el7.centos.x86_64 conflicts abrt
        Feb 08 18:57:52 ip-172-30-0-197 sudo[722]: --> Finished Dependency Resolution
        Feb 08 18:57:52 ip-172-30-0-197 sudo[722]: Error: scylla-server conflicts with abrt-2.1.11-36.el7.centos.x86_64
        Feb 08 18:57:52 ip-172-30-0-197 sudo[722]: You could try using --skip-broken to work around the problem
        Feb 08 18:57:53 ip-172-30-0-197 sudo[722]: You could try running: rpm -Va --nofiles --nodigest
        Feb 08 18:57:53 ip-172-30-0-197 systemd[1]: scylla-server.service: control process exited, code=exited status=1
        Feb 08 18:57:53 ip-172-30-0-197 systemd[1]: Failed to start Scylla Server.
        Feb 08 18:57:53 ip-172-30-0-197 systemd[1]: Unit scylla-server.service entered failed state.
        A
        Feb 08 18:57:53 ip-172-30-0-197 systemd[1]: scylla-server.service failed.V

Abrt should be removed from the AMI at creation time.

**Comment by dorlaor** [comment]
Can you please check why it conflicts? Maybe there is a common file both
RPMs use

**Comment by slivne** [comment]
It conflicts becasue we say it does

commit 5270da1eef272082f0c2cf7829566d865e620270
Author: Takuya ASADA <syuu@scylladb.com>
Date:   Thu Jan 14 04:24:41 2016 +0900

    dist: prevent use abrt with scylla
    
    scylla should use with systemd-coredump, not abrt.
    Fixes #762
    
    Signed-off-by: Takuya ASADA <syuu@scylladb.com>
    Message-Id: <1452713081-32492-1-git-send-email-syuu@scylladb.com>
```
diff --git a/dist/redhat/scylla-server.spec.in b/dist/redhat/scylla-server.spec.in
index 146ade6..2ec6091 100644
--- a/dist/redhat/scylla-server.spec.in
+++ b/dist/redhat/scylla-server.spec.in
@@ -12,6 +12,7 @@ BuildRequires:  libaio-devel libstdc++-devel cryptopp-devel hwloc-devel numactl-
 %{?fedora:BuildRequires: boost-devel ninja-build ragel antlr3-tool antlr3-C++-devel python3 gcc-c++ libasan libubsan}
 %{?rhel:BuildRequires: scylla-libstdc++-static scylla-boost-devel scylla-ninja-build scylla-ragel scylla-antlr3-tool scylla-antlr3-C++-devel python34 scylla-gcc-c++ >= 5.1.1}
 Requires:       systemd-libs xfsprogs mdadm hwloc
+Conflicts:      abrt
```

It is already resolved on the AMI

commit b5029dae7eaf745d5f23831bb7996933b4f714e9
Author: Takuya ASADA <syuu@scylladb.com>
Date:   Thu Jan 21 17:47:59 2016 -0500

    dist: remove abrt from AMI, since it's not able to work with Scylla
    
    New CentOS Base Image contains abrt by default, so remove it.
    
    Signed-off-by: Takuya ASADA <syuu@scylladb.com>
    Message-Id: <1453416479-28553-2-git-send-email-syuu@scylladb.com>

```
diff --git a/dist/ami/build_ami.sh b/dist/ami/build_ami.sh
index c921ee6..ccdd7dd 100755
--- a/dist/ami/build_ami.sh
+++ b/dist/ami/build_ami.sh
@@ -38,9 +38,9 @@ if [ ! -d packer ]; then
 fi
 
 if [ $LOCALRPM = 0 ]; then
-    echo "sudo sh -x -e /home/centos/scylla_install_pkg; sudo sh -x -e /usr/lib/scylla/scylla_setup -a" > scylla_deploy.sh
+    echo "sudo yum remove -y abrt; sudo sh -x -e /home/centos/scylla_install_pkg; sudo sh -x -e /usr/lib/scylla/scylla_setup -a" > scylla_deploy.sh
 else
-    echo "sudo sh -x -e /home/centos/scylla_install_pkg -l /home/centos; sudo sh -x -e /usr/lib/scylla/scylla_setup -a" > scylla_deploy.sh
+    echo "sudo yum remove -y abrt; sudo sh -x -e /home/centos/scylla_install_pkg -l /home/centos; sudo sh -x -e /usr/lib/scylla/scylla_setup -a" > scylla_deploy.sh
 
 fi
```

**Comment by dorlaor** [comment]
It's not a big deal but isn't there a better way to prevent arbt to report
a buggy scylla?

Check the docs:

BlackListedPaths = /usr/share/doc/*, */example*
Crashes in executables in these paths will be ignored by *ABRT*.
